src/base-command.ts

```diff
@@ -92,11 +99,98 @@ export default abstract class BaseCommand extends Command {
     return super.parse(options, [...args, ...flags]);
   }
 
+  async _getFilteredMetadata(calling_class: any): Promise<any> {
```

Comment:
Maybe `_getFilteredSentryMetadata` so we know it's just for Sentry

Reply:
updated function name